### DISCUSSION TITLE
Gene Labels in browser plot unreadable

### DISCUSSION THREAD

**Discussion by sylestiel** [GreenleafLab/ArchR#855] [General]
Hi,

 How can I remove the grey background in the gene labels when plotting the browser Tracks. It is almost impossible to read the gene names. I'm not sure what changed for it to render like this.
<img width="526" alt="Screen Shot 2021-06-29 at 9 05 51 AM" src="https://user-images.githubusercontent.com/64283689/123812805-02f41780-d8ba-11eb-8b61-ab24e6b7f719.png">
 
Can you give me some suggestions to fix this problem?

Thanks!

**Comment by rcorces** [comment]
No idea. Thats something in your environment - maybe ggrepel or ggplot2. That is not a normal behavior.

**Comment by sylestiel** [comment]
What version of ggrepel or ggplot is normally supported by ArchR. Should I just reload the entire ArchR Suite?

**Reply by rcorces** [reply]
https://www.archrproject.com/bookdown/session-information.html

**Comment by sylestiel** [comment]
<img width="880" alt="Screen Shot 2021-06-29 at 10 16 30 AM" src="https://user-images.githubusercontent.com/64283689/123825260-62efbb80-d8c4-11eb-8462-245688439172.png">

I reloaded ArchR. Attached is the sessionInfo().
Can you help me spot the problem.

**Reply by rcorces** [reply]
I have no way of helping here. sorry.
